### PR TITLE
chore(demo-app): message content update

### PR DESCRIPTION
**Pull request by simeng-li**
message content update

## Summary

Update the demo app notification message content
image

## Testing

test locally
@eng

### CODE REVIEW

File: packages/phrases/src/locales/en/translation/demo-app.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const demo_app = {
-  notification: 'Use your admin console account to sign in to this demo app.',
+  notification:
+    "You will need an admin console account to sign in to the demo app. If you don't already have an account, create one for free.",
```
[the remark this answers is not in the document]
Comment (demonzoo) [reply packages/phrases/src/locales/en/translation/demo-app.ts:3]: Idk, but `create one for free` sounds to me like we have a paid version to create more than one. 😂

File: packages/phrases/src/locales/zh-cn/translation/demo-app.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const demo_app = {
-  notification: '使用管理控制台帐号登录示例应用',
+  notification:
+    '管理控制台的用户名和密码是 demo app 的默认登录方式。点击下方创建账号或用现有账号登录。',
```
Comment (demonzoo) [inline packages/phrases/src/locales/zh-cn/translation/demo-app.ts:3]: I like the Chinese translated version. Can we align the English version to this?

File: packages/phrases/src/locales/en/translation/demo-app.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (simeng-li) [reply packages/phrases/src/locales/en/translation/demo-app.ts:3]: Need to follow up with betty.

[the remark this answers is not in the document]
Comment (simeng-li) [reply packages/phrases/src/locales/en/translation/demo-app.ts]: updated

**Review by demonzoo — APPROVED** [review]

**Pull request merged**